.vscode/settings.json

```diff
@@ -3,5 +3,14 @@
     "**/node_modules": true,
     "**/bower_components": true,
     "**/*.code-search": true
-  }
+  },
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[typescript]": {
+    "editor.defaultFormatter": "biomejs.biome"
+  },
+  "[javascript]": {
```

Comment:
Was too fast with merging.

`[css]` and JSON could be added too, since there is support for them as well: https://biomejs.dev/internals/language-support/

Or maybe for

Reply:
Will have a look